paltas/Analysis/dataset_generation.py

```diff
@@ -120,6 +120,37 @@ def unnormalize_outputs(input_norm_path,learning_params,mean,standard_dev=None,
 		if cov_mat is not None:
 			cov_mat[:,lpi,:] *= param_std
 			cov_mat[:,:,lpi] *= param_std
+			
+# TODO: write test after moving (make sure identity operation w/ unnormalized)
+def normalize_mu_prec(mu,prec_mat,input_norm_path):
+    """Helper function to convert mu, prec_matrix to normalized parameter 
+        space
+        
+    Args:
```

Comment:
I'm not familiar with this string formatting. Does that shape information translate to the read the docs?

Remove the indentation from Returns (should have same indentation as Args)